scipy/stats/_continuous_distns.py

```diff
@@ -3247,6 +3247,142 @@ def _ppf(self, q, k, s):
 mielke = mielke_gen(a=0.0, name='mielke')
 
 
+class kappa4_gen(rv_continuous):
+    """Kappa 4 parameter distribution
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for kappa4 is::
+
+        kappa4.pdf(x, h, k) = (1.0 - k*x)**(1.0/k - 1.0)*
+                              (1.0 - h*(1.0 - k*x)**(1.0/k))**(1.0/h-1)
+
+    if ``h`` and ``k`` are not equal to 0.
```

Comment:
I think it also needs formulas for special cases, `h` or `k` being zero. To simplify formulas, you might also want to list cdfs not pdfs: eqs (1-4) in Finney look simpler than (11-14).

Domains of support should also be listed here.